src/components/consul/detail/ConsulVoting.js

```diff
@@ -80,18 +80,18 @@ export const ConsulVotingComponent = ({ votesData, onRefresh, id }) => {
   );
 };
 
-ConsulVotingComponent.propTypes = {
-  votesData: PropTypes.object.isRequired,
-  id: PropTypes.string,
-  onRefresh: PropTypes.func
-};
-
 const styles = StyleSheet.create({
+  icon: {
+    paddingHorizontal: 10
+  },
   iconButton: {
     alignItems: 'center',
     marginHorizontal: 10
```

Comment:
do we need normalization here? or is it ok not having it in this particular place.

Reply:
I have tested. Better to use. It is not understandable unless you are very careful with devices of different sizes. But still there is a slight difference.

Reply:
8d1121aeab4ff33f5e37bc5f2568faec55c0b11f